README.md

```diff
@@ -302,7 +302,7 @@ Once ArgoCD and Kubernetes are configured, you can then set the required environ
 VAULT_ADDR: Your HashiCorp Vault Address
 TYPE: vault
 AUTH_TYPE: k8s
-K8S_MOUNT_POINT: Mount Point of your kubernetes Auth
+K8S_MOUNT_PATH: Mount PATH of your kubernetes Auth (optional)
```

Comment:
Why is PATH in all caps in the description?

Reply:
Mistake, will fix